docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,191 @@
+This XKCD comic feels always relevant when talking about dependencies : <https://xkcd.com/2347/>
+
+Each software dependency is a risk. For example: SQL Injections, is the ORM you are using properly escaping the content you are passing to it? Will your current driver be able to connect to your database if it gets upgraded ?
+Every dependency, while solving an issue for you, will also bring new risks to your software. How can you mitigate those risks ?
+
+If the title of the article is of any help, the idea would be to keep your dependencies up-to-date to fix bugs, security issues, and more.
+
+## I don’t feel like upgrading, what if …
+
+But what if the upgrades do the complete opposite? What if the new version introduces a new security issue?
+
+For example, a new version of a dependency could be victim of a supply chain attack. If you never update your dependencies, you won’t get the infected version. Sounds fair, right ?
+
+It’s a valid argument, but your application doesn’t exist in a vacuum; many external factors could force you to upgrade at some point, and when it happens it’s usually at the wrong time.
+
+Let’s go through some examples
+
+### An external factor has changed, forcing you to change your app
+
+Let’s say the machine your apps runs on is obsolete and you need to move to another machine. This could be as simple as “install app on machine; done” but usually ends up with:
+
+1. The newer OS version prevents you from installing an outdated runtime
+1. You need to find a compatible runtime for your new hardware
+1. Your app’s dependencies aren’t compatible with your new runtime, you need to update them as well
+
+Instead of a single change, you now have a combination of changes, each can go wrong in a different way.
+
+### A new vulnerability is found on a library you depend on
+
+As we’ve covered at the beginning of the article this can happen and can be an all-hands-on-deck kind of situation. Imagine a legacy application; continuous integration is constantly failing — or worse it may not even exist. How long do you think it will take to deploy that single library update ?
+
+### The team already has too many dependencies in too many different versions
+
+That’s unfortunate, but it will happen if your team or company exists for long enough. Upgrading libraries is not only about bumping versions, sometimes it’s also about replacing a library by another library.
+
+## How do you see your dependencies ?
+
+Look at your software as if it was a train. You are the locomotive and each wagon is a dependency, how long would your train be ? Locomotives can pull dozens, even hundreds of wagons, but they eventually reach a limit. The situation is the same for your brain, each dependency adds to your cognitive load, at some point it’s just too much. You are using three different libraries for caching? There is a chance you need to know how each library works, and your present and future teammates might have to know that too.
```

Comment:
```suggestion
Look at your software as if it was a train.
You are the locomotive and each wagon is a dependency, how long would your train be?
Locomotives can pull dozens, even hundreds of wagons, but they eventually reach a limit.
The situation is the same for your brain, each dependency adds to your cognitive load, at some point it’s just too much.
You are using three different libraries for caching?
There is a chance you need to know how each library works, and your present and future teammates might have to know that too.
```

- One sentence per line
- Remove whitespace between line ending and `?` character